ci/github/system-testing/environment-setup.bash

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -euo pipefail
+IFS=$'\n\t'
+
+install() {
+    bash ci/helpers/ensure_python_pip.bash
+    pushd tests/environment-setup; pip3 install -r requirements/ci.txt; popd
+    make .env
+}
+
+test() {
+    pytest --color=yes -v tests/environment-setup
```

Comment:
minor: just wondering if we should not have a makefile recipe here such as ```make ci-tests``` or so? this could make it a bit easier to try on in local development?